widget/embedded/src/pages/ThemePage.tsx

```diff
@@ -0,0 +1,42 @@
+import { i18n } from '@lingui/core';
```

Comment:
This page should be matched with Figma, I will review this file after that.

Reply:
I have done this in  sub setting-page PR